mentorship iosTests/MembersTests.swift

```diff
@@ -82,7 +82,7 @@ class MembersTests: XCTestCase {
         let expectation = XCTestExpectation(description: "response")
         membersService.sendRequest(menteeID: 0, mentorID: 1, endDate: 0, notes: "") { resp in
             XCTAssertEqual(resp.message, mockJSON.message)
-            XCTAssertEqual(resp.success, true)
+            XCTAssertEqual(resp.success, resp.message == "Mentorship relation was sent successfully.")
```

Comment:
We have an assert statement for the message, so this doesn't really adds any value...
Just remove the line.